mara-ros2.repos

```diff
@@ -2,20 +2,36 @@ repositories:
   mara:
     type: git
     url: https://github.com/acutronicrobotics/MARA.git
-    version: master
+    version: dashing
   HRIM:
     type: git
     url: https://github.com/erlerobot/HRIM.git
     version: Coliza
-  mara_examples:
-    type: git
-    url: https://github.com/AcutronicRobotics/mara_examples.git
-    version: master
   robotiq_modular_gripper:
     type: git
     url: https://github.com/AcutronicRobotics/robotiq_modular_gripper.git
     version: master
   hans_modular_servomotor:
     type: git
     url: https://github.com/AcutronicRobotics/hans_modular_servomotor.git
-    version: master
\ No newline at end of file
+    version: master
+  gazebo_ros_pkgs:
+      type: git
+      url: https://github.com/ros-simulation/gazebo_ros_pkgs
+      version: ros2
+  control_msgs:
+    type: git
+    url: https://github.com/erlerobot/control_msgs/
+    version: crystal
+  geometry2:
+    type: git
+    url: https://github.com/acutronicrobotics/geometry2/
+    version: ros2
```

Comment:
I have had to remove geometry2 repository in order to be able to compile it today. Everything else has worked fine.
I have dashing-desktop installed.

Reply:
need to test this as well. I plan to move to dashing today and see which repos are working